broker/lib/bosh/BoshDirectorClient.js

```diff
@@ -621,24 +621,37 @@ class BoshDirectorClient extends HttpClient {
 
   /*  Task operations */
 
-  getTasks(options) {
+  getTasks(options, fetchDirectorForDeployment) {
     const query = _.assign({
       limit: 1000
```

Comment:
should it be hard-coded or config driven ?or  you can put it in constant. Just give a think. If you strongly feel it should be like this. It's fine.